boards/arm/gd32e103v_eval/gd32e103v_eval.dts

```diff
@@ -102,6 +102,7 @@
 
 &timer0 {
 	status = "okay";
+	prescaler = <128>;
```

Comment:
Hi @feilongfl ,

LGTM! I only think you can rework commit message. You need a prescaler to have long time period, right?